### ISSUE TITLE
MacOS release version doesn't start

### ISSUE THREAD

**Issue by inikolaev** [jmacdonald/amp#49]
Have just downloaded MacOS version from releases and getting the following error when trying to start it:

```bash
dyld: Library not loaded: /usr/local/opt/libssh2/lib/libssh2.1.dylib
  Referenced from: /Users/inikolaev/downloads/amp
  Reason: image not found
Abort trap: 6
```

Am I expected to install `libssh2` or should it be working out-of-the-box? Wouldn't like to install anything though.

**Comment by jmacdonald** [comment]
Yep, it looks like you'll need `libssh2`, too. Statically linked binaries are the next hurdle, but we're not there yet; that'd prevent you from having to install anything else.

Taking this as a note to include `libssh2` as a dependency for now; I'll follow up with another issue to investigate static linking.

**Comment by jmacdonald** [comment]
`libssh2` is a specified dependency in the Homebrew formula, so yes, you'll need to install that (as well as `openssl`). As soon as the [homebrew-core PR](https://github.com/Homebrew/homebrew-core/pull/22441) is merged, this will all be automated.
